### PR TITLE
Only allow some characters within a request URL to PyPi.

### PR DESCRIPTION
**Pull request by Xithrius**
If someone spams a bunch of characters, we don't want to send a request to PyPi with a bunch of gibberish. This also protects against injections (if possible, if any).

Some examples of cases with illegal characters within them:
image

### CODE REVIEW

**Comment by Xithrius** [comment]
There are some changes from a different branch on here. Fixing.

File: bot/exts/info/pypi.py [hunk 1]
```diff
@@ -32,6 +36,11 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
         )
         embed.set_thumbnail(url=PYPI_ICON)
 
+        if (character := re.search(ILLEGAL_CHARACTERS, package)) is not None:
+            embed.description = f"Illegal character passed into command: '{escape_markdown(character.group(0))}'"
```
Comment (Shivansh-007) [inline bot/exts/info/pypi.py:40]: Missing a `.` full stop at the end of the sentence,

```suggestion
            embed.description = f"Illegal character passed into command: '{escape_markdown(character.group(0))}'."
```

**Review by Shivansh-007 — APPROVED** [review]
Thanks! Looks Good To Me! Just one thing, not sure if it is needed or not tho.

Tested locally.

File: bot/exts/info/pypi.py (hunk 1 above)
Comment (Xithrius) [reply bot/exts/info/pypi.py:40]: I tested this a while ago. I didn't think it looked right. Here's a comparison.

With period:
image

And without:
image

Comment (Shivansh-007) [reply bot/exts/info/pypi.py:40]: Yeah, even I had the same feeling.

**Review by ChrisLovering — APPROVED** [review]
All looks good to me!

**Review by scragly — APPROVED** [review]

**Pull request merged**